torch/compiler/__init__.py

```diff
@@ -118,3 +119,31 @@ def rand_foo():
     from torch._inductor import cudagraph_trees
 
     cudagraph_trees.mark_step_begin()
+
+def wrap_numpy(fn):
+    r"""Decorator that turns a function from ``np.ndarray``s to ``np.ndarray``s into a function
+    from ``torch.Tensor``s to ``torch.Tensor``s.
+
+    It is designed to be used with :func:`torch.compile`. It allows to compile a NumPy function
+    as if it were a PyTorch function. If the function can be traced without graph breaks,
+    this allows you to run NumPy code on CUDA.
+    Similarly, you can pass Tensors with ``requires_grad=True`` and then call ``backward()`` on the output
```

Comment:
nit to shorten the doc, this can be folded in the sentence above: `code on CUDA and with gradients enabled.`